src/actor/monster/sheets-tabs/main-tab.hbs

```diff
@@ -0,0 +1,161 @@
+<div class="main-tab">
```

Comment:
NB: I'm really interested in avoiding accidental divergence between this file and the character one. Once this is merged, I'd love to see us DRY this up with partials, because it's hard to see at a glance what is different between characters/monsters, but would be much easier if things were something like
```
<div class="left-content">
  {{> "points_and_defense_partial_path" }}
  {{> "affinities_partial_path" }}
</div>
```

Reply:
Yah makes total sense